src/dbnode/storage/namespace.go

```diff
@@ -1700,9 +1705,11 @@ func (n *dbNamespace) aggregateTiles(
 
 	n.log.Info("finished large tiles aggregation for namespace",
 		zap.String("sourceNs", sourceNs.ID().String()),
-		zap.Int("shards", len(targetShards)),
+		zap.Time("targetBlockStart", targetBlockStart),
+		zap.Time("lastSourceBlockEnd", lastSourceBlockEnd),
```

Comment:
nit; `sourceBlockEnd`? `end` implies `last` 👍

Reply:
Well, there can be multiple source blocks being aggregated, wanted to emphasize that this is the end of the last of them.